sentry/templates/ingress.yaml

```diff
@@ -22,6 +22,19 @@ spec:
             backend:
               serviceName: {{ template "sentry.fullname" . }}-nginx
               servicePort: {{ .Values.nginx.service.port }}
+    {{- else if eq (default "nginx" .Values.ingress.regexPathStyle) "aws-alb" }}
```

Comment:
Wouldn't that be suitable for all cases (alb, traeffic,..) ?
I think the paths defined in that order remove the need for any RegEx.

Reply:
Not sure about how other ingress controller process the paths.
AWS alb is very ... limited, it process it in the configuration order but if nginx or traefik process it from more restrictive to less restrictive, this *could* not work as expected. 
And the main problem is still here: Each ingress controller has it's own regex syntax so anyway we have to deal with it.